addEventInterviewer/index.ts

```diff
@@ -0,0 +1,66 @@
+import { AzureFunction, Context, HttpRequest } from '@azure/functions';
+import getConnection from '../shared/services/db';
+import { httpErrorHandler, ObjectNotFoundFunctionError, ValidationFunctionError, FunctionError } from '../shared/error/error';
+import EventModel from '../shared/event/event.model';
+import NodeModel from '../shared/node/node.model';
+import UserModel from '../shared/user/user.model';
+import addEventInterviewerSchema from './joi';
+import HttpTriggerResult from '../shared/httpTriggerResult';
+import { Role, User } from '../shared/user/user.interface';
+import authWrapper from '../shared/authWrapper';
+import nodegroupModel from '../shared/nodegroup/nodegroup.model';
+
+const functionName = 'addEventInterviewer';
+
+const addEventInterviewer: AzureFunction = async (context: Context, req: HttpRequest, user: User): Promise<HttpTriggerResult> => {
+    try {
+        if (user.role !== Role.RamadIturOfUnit && user.role !== Role.RamadIturAssistant) {
+            throw new FunctionError(functionName, 'Cannot get events for now');
+        }
+
+        const { error } = addEventInterviewerSchema.validate(req);
+        if (error) {
+            throw new ValidationFunctionError(functionName, error.message);
+        }
+
+        await getConnection();
+
+        const { eventId, userId } = context.bindingData;
+        const interviewerUser = await UserModel.findById(userId).exec();
+        if (interviewerUser.role !== Role.Interviewer) {
+            throw new ValidationFunctionError(functionName, `Role of user is not interviewer.`);
+        }
+        const event = await EventModel.findById(eventId).exec();
+        if (!event) throw new ObjectNotFoundFunctionError(functionName, 'Event', eventId);
+        if (!interviewerUser) throw new ObjectNotFoundFunctionError(functionName, 'User', userId);
+
+        const eventNode = await NodeModel.findById(event.node.id).exec();
+        const eventNodeGroup = await nodegroupModel.findById(eventNode.nodeGroupId).exec();
+
+        if (!eventNodeGroup.usersIds.includes(interviewerUser._id)) {
+            throw new FunctionError(functionName, `Interviewer is not in the NodeGroup of the event`);
+        }
+
+        if (user.role === Role.RamadIturOfUnit) {
+            if (eventNode.unitId.toString() !== user.unitId?.toString()) {
```

Comment:
```suggestion
            if (eventNode.unitId?.toString() !== user.unitId?.toString()) {
```

You did it on `getEvent`